script.module.resolveurl/lib/resolveurl/plugins/allviid.py

```diff
@@ -17,12 +17,20 @@
 """
 
 from resolveurl.plugins.__resolve_generic__ import ResolveGeneric
-
+from resolveurl.lib import helpers
 
 class AllViidResolver(ResolveGeneric):
     name = 'AllViid'
     domains = ['allviid.xyz', 'all-vid.autos']
-    pattern = r'(?://|\.)(all-?vi*d\.(?:xyz|autos))/(?:embed-)?([0-9a-zA-Z]+)'
+    pattern = r'(?://|\.)(.*?all-?vi*d\.(?:xyz|autos))/(?:embed-)?([0-9a-zA-Z]+)'
```

Comment:
why add .*? in front of `all-`?

Reply:
Because the domain I want to add is 'i.all-vid.autos'